OneBusAway/ui/trip_details/OBAArrivalAndDepartureViewController.m

```diff
@@ -113,38 +115,33 @@ - (instancetype)initWithArrivalAndDepartureConvertible:(NSObject<OBAArrivalAndDe
 
 - (void)viewDidLoad {
     [super viewDidLoad];
-
+    
```

Comment:
watch these whitespace-only lines please. Xcode has an option to automatically truncate them that I highly recommend enabling.
![screen-shot-2017-02-15-at-10 47 37-pm](https://cloud.githubusercontent.com/assets/2254/23010554/e5b06656-f3d0-11e6-896b-09cd33ef63ce.jpg)